components/odh-notebook-controller/controllers/notebook_feast_config.go

```diff
@@ -0,0 +1,153 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	nbv1 "github.com/kubeflow/kubeflow/components/notebook-controller/api/v1"
+	corev1 "k8s.io/api/core/v1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/utils/ptr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	feastConfigMapSuffix  = "-feast-config"
+	feastConfigVolumeName = "odh-feast-config"
+	feastConfigMountPath  = "/opt/app-root/config/feast"
```

Comment:
```suggestion
	feastConfigMountPath  = "/opt/app-root/src/config/feast"
```

we **DO** want to mount this feast config volume **within** the `/opt/app-root/src` volume.

Reply:
Acknowledge, adjusting to the code to use `/opt/app-roo/src/feast-config`
once i do the full testing, will adjust to `/opt/app-root/src/config/feast`